libs/wire-subsystems/src/Wire/UserGroupStore/Postgres.hs

```diff
@@ -220,6 +221,94 @@ getUserGroup team id_ includeChannels = do
             where ug.id = ($1 :: uuid) and ug.team_id = ($2 :: uuid)
           |]
 
+getUserGroupsWithMembers ::
+  forall r.
+  ( UserGroupStorePostgresEffectConstraints r
+  ) =>
+  UserGroupPageRequest ->
+  Sem r UserGroupPageWithMembers
+getUserGroupsWithMembers req = runTransaction TxSessions.ReadCommitted TxSessions.Read $ do
+  groups :: [UserGroup] <- Tx.statement () $ refineResult (mapM toUserGroup) $ buildStatement query rows
+  count <- getUserGroupCount req
+  return $ UserGroupPage groups count
+  where
+    rows :: HD.Result [(UUID, Text, Int32, UTCTime, Vector UUID, Int32)]
+    rows = HD.rowList $
+      (,,,,,)
+        <$> HD.column (HD.nonNullable HD.uuid)
+        <*> HD.column (HD.nonNullable HD.text)
+        <*> HD.column (HD.nonNullable HD.int4)
+        <*> HD.column (HD.nonNullable HD.timestamptz)
+        <*> decodeUuidVector
+        <*> HD.column (HD.nonNullable HD.int4)
+
+    query :: QueryFragment
+    query = mconcat $ map literal [
+      "select", T.intercalate ", " [
+        "g.id :: uuid",
+        "g.name :: text",
+        "g.managed_by :: int",
+        "g.created_at :: timestamptz",
+        "coalesce(array_agg(gm.user_id), array[]::uuid[]) :: uuid[]",
+        "count(*) :: int"
+        ],
+      "from user_group g",
+      "left join user_group_member gm on g.id = gm.user_group_id"
+      ] <> [where_ (groupMatchIdName req <> groupPaginationWhereClause req)] <> map literal [
+      "group by g.team_id, g.id"
```

Comment:
The query selects more columns than in the group by. Not sure how this can work.

Reply:
ok, I see, thanks!